src/CalcManager/CalcException.h

```diff
@@ -15,12 +15,3 @@ class CalcException : std::exception
 private:
     HRESULT m_hr;
 };
-
-void IFT(HRESULT hr)
```

Comment:
This was moved into `History.cpp`, the only consumer of `CalcException` is also in there, should I move it into `History.cpp` as well and remove this file?

Reply:
Yep, I think that's a good change.  Let's limit visibility of functions to places where they are needed.

Reply:
Updated